src/screens/Home/HomeDetails.tsx

```diff
@@ -0,0 +1,77 @@
+import Image from "next/image";
+import Link from "next/link";
+import { Product } from "../../components/ProductCard/types";
+import {
+  Banner,
+  Text,
+  BannerTextContent,
+  BannerTitle,
+  ContentContainer,
+  ProductDisplay,
+  ProductDisplayTextContent,
+  ProductDisplayTitle,
+} from "./HomeDetails.styled";
+import { ProductCard } from "../../components/ProductCard/ProductCard";
+import {
+  BANNER_TEXT,
+  BANNER_TITLE,
+  MOST_WANTED,
+  MOST_WANTED_COMPLEMENT,
+} from "./constants";
+
+export type HomeDetailsProps = {
+  products: Product[];
+};
+
+export const HomeDetails = ({ products }: HomeDetailsProps) => {
+  return (
+    <>
+      <ContentContainer>
+        <Banner>
+          <Image
+            src="https://images.unsplash.com/photo-1623625434531-d130448273c1?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=1471&q=80"
+            data-testid="banner"
+            width={800}
+            height={400}
+            alt="banner"
+            className="banner"
+          />
+          <BannerTextContent>
+            <BannerTitle>{BANNER_TITLE}</BannerTitle>
+            <Text>{BANNER_TEXT}</Text>
+            <Link href="/">
+              <Image
+                src="https://cdn-icons-png.flaticon.com/128/318/318476.png"
+                data-testid="navigate-icon"
+                width={30}
+                height={30}
+                alt="navigate-icon"
+              />
+            </Link>
+          </BannerTextContent>
+        </Banner>
+        <ProductDisplay>
+          <ProductDisplayTextContent>
+            <ProductDisplayTitle>{MOST_WANTED}</ProductDisplayTitle>
+            <Text>{MOST_WANTED_COMPLEMENT}</Text>
+          </ProductDisplayTextContent>
+          {products.map(
+            (product, index) =>
+              index <= 3 && <ProductCard key={index} product={product} />
```

Comment:
you could use a constant here to describe this number
or you could use `products.slice(0, 3)` to get only the ones you need